ios/REAModule.mm

```diff
@@ -20,6 +20,8 @@
 #import <RNReanimated/REAModule.h>
 #import <RNReanimated/REANodesManager.h>
 
+#import "SingleInstanceChecker.h"
```

Comment:
Oops, I totally forgot that we should use angle brackets here. I will prepare a PR that fixes it and also adds CI rule.